src/BenchmarkDotNet/Templates/MonoAOTLLVMCsProj.txt

```diff
@@ -20,6 +20,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <StartupObject>BenchmarkDotNet.Autogenerated.UniqueProgramName</StartupObject>
     <SelfContained>true</SelfContained>
+    <!-- fix System.Runtime.CompilerServices.Unsafe doesn't support netcoreapp2.0 -->
+    <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
```

Comment:
this project file should never be used for such old tfms (because Mono was not supported back then)

Reply:
It was added in net6.0, which is now out of support. 9.0 nuget packages will trigger the warning.